### ISSUE TITLE
App rejection

### ISSUE THREAD

**Issue by futuretap** [tonymillion/Reachability#96]
My app was just rejected:

> **Non-public API usage:**
> The app contains or inherits from non-public classes in WhereTo: Reachability

So I guess naming the class `Reachability` is banned by Apple. :-(

**Comment by g0ld2k** [comment]
I assume you are using CocoaPods 0.36 and the use_frameworks! flag in your Podfile?  If so this is the same as #95

**Comment by futuretap** [comment]
I don't use `use_frameworks!`. However, CocoaPods are included in my own framework that is used to share code between my app and my WatchKit extension.

**Comment by blakeperdue** [comment]
So, what's the deal @futuretap? Can you still rename the class and get approved?

**Comment by futuretap** [comment]
Yeah, renaming the class works fine and the app got approved multiple times already.

**Comment by blakeperdue** [comment]
Great, thanks for the update. @tonymillion I think you can close this issue.

**Comment by futuretap** [comment]
I still trink that renaming the framework would help a lot because it would allow integrating it as-is via CocoaPods.

**Comment by blakeperdue** [comment]
I'm going to submit without renaming, without using `use_frameworks!` and without CocoaPods. Will report what happens.

**Comment by futuretap** [comment]
Then the API checker most likely won't find anything. Our app began to be rejected only when we migrated Reachability into a framework.

**Comment by blakeperdue** [comment]
My app was approved without renaming just as @futuretap suggested.
